src/main/java/org/isf/security/UserDetailsServiceImpl.java

```diff
@@ -57,7 +62,10 @@ public UserDetails loadUserByUsername(String username) throws UsernameNotFoundEx
             throw new UsernameNotFoundException(username + " was not found");
         }
         List<SimpleGrantedAuthority> authorities = new ArrayList<>();
-        authorities.add(new SimpleGrantedAuthority(user.getUserGroupName().getCode()));
+        List<Permission> permissions = permissionManager.retrievePermissionsByUsername(username);
+        for (Permission p : permissions) {
+        	  authorities.add(new SimpleGrantedAuthority(p.getName()));	
```

Comment:
this seems to be indented too far in

Reply:
please on my side i don't have this issue
![error1](https://github.com/informatici/openhospital-api/assets/101590821/62a48e8b-7bdb-43e1-90a3-29fab1264e03)